packages/vue/src/server/index.js

```diff
@@ -472,14 +497,17 @@ export default function initReactivesearch(componentCollection, searchState, set
 										};
 									}
 									timestamp[component] = res._timestamp;
+									const hitsObj = response.hits
```

Comment:
@mohdashraf010897 This change is not required, which seems specific to your endpoint example.